src/pages/operations/OperationList.tsx

```diff
@@ -84,6 +79,9 @@ const OperationList: FC = () => {
               <div className="u-truncate u-text--muted">
                 <OperationInstanceName operation={operation} />
               </div>
+              <div className="u-text--muted">
+                Project name: {getProjectName(operation)}
```

Comment:
```suggestion
                Project: {getProjectName(operation)}
```
I think we can be a bit more concise here.

Reply:
Yeah concise looks better